### ISSUE TITLE
[BUG] - r2modman will not run without special launch arg on linux

### ISSUE THREAD

**Issue by TrainDoctor** [ebkr/r2modmanPlus#658]
**Describe the bug**
r2mm will crash with the included message when running on openSUSE Linux.
This is unless the ``--no-sandbox`` launch argument  is used when launching r2modman.

**To Reproduce**
1. Download the RPM package version of r2modman.
2. After the .rpm is fully downloaded open with "YaST Software".
2a.  if running KDE, it will default to "Discover" which is not what we want to try to install this program with.
3. After an auto-refresh and other steps, you will be prompted with "nothing provides 'libuuid' needed by ... r2modman-3.1.20-1.x86_64"
4. Force install by breaking and ignoring dependencies, r2mm will install but will crash shortly after trying to launch.
5. Find r2modman in the application launcher and right click, selecting "Edit Application".
6. For launching:
6a. Now edit the application launch param to reflect this: ``/opt/r2modman/r2modman --no-sandbox %U``
6b. (or if running from console simply include ``--no-sandbox`` to the end after typing ``r2modman``.

**Expected behavior**
r2modman launches from the Application Launcher.

**Additional context**
I am running openSUSE Tumbleweed.
```
Linux localhost.localdomain 5.14.14-1-default #1 SMP Thu Oct 21 05:05:03 UTC 2021 (2b5383f) x86_64 x86_64 x86_64 GNU/Linux
LSB Version:    n/a
Distributor ID: openSUSE
Description:    openSUSE Tumbleweed
Release:        20211030
Codename:       n/a
```

**Comment by ebkr** [comment]
This may be fixed in 3.1.29?

Just upgraded Electron by quite a few versions, so it could have just been an old problem.

**Comment by TrainDoctor** [comment]
I'll give it a shot as soon as I can. Thanks ebkr.

**Comment by TrainDoctor** [comment]
> This may be fixed in 3.1.29?
> 
> Just upgraded Electron by quite a few versions, so it could have just been an old problem.

Works from the pacman file like a charm. I recommend that that file be packaged as a .pkg.tar.zst suffix as that is the standard packaging convention for arch. 👍🏻
